### PR TITLE
rebase azure-function with main

### PR DESCRIPTION
**Pull request by nooreldeensalah**

### CODE REVIEW

File: HttpTrigger1/main.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+
+from .models import Label, Participant, Meeting, create_tables
+
+
+def main(req: func.HttpRequest) -> func.HttpResponse:
+    logging.info("Python HTTP trigger function processed a request.")
+    request_body = req.get_json()
+    subprocess.run("git clone -b gh-pages --single-branch https://github.com/ElGarash/Jitsi-Meetings-Dashboard HttpTrigger1/gpages")
+    os.chdir(".\\HttpTrigger1\\gpages")
```
Comment (AbdulrhmnGhanem) [inline HttpTrigger1/main.py:15]: You can use pathlib here.

File: HttpTrigger1/main.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+from datetime import datetime
+
+
+from .models import Label, Participant, Meeting, create_tables
+
+
+def main(req: func.HttpRequest) -> func.HttpResponse:
+    logging.info("Python HTTP trigger function processed a request.")
+    request_body = req.get_json()
+    subprocess.run("git clone -b gh-pages --single-branch https://github.com/ElGarash/Jitsi-Meetings-Dashboard HttpTrigger1/gpages")
```
Comment (AbdulrhmnGhanem) [inline HttpTrigger1/main.py:14]: use the `database_location` instead of hardcoding its `HttpTrigger1/gpages`.

File: .vscode/tasks.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+			"windows": {
+				"command": "${config:azureFunctions.pythonVenv}\\Scripts\\python -m pip install -r requirements.txt"
+			},
+			"linux": {
+				"command": "${config:azureFunctions.pythonVenv}/bin/python -m pip install -r requirements.txt"
+			},
+			"problemMatcher": []
+		}
+	]
+}
```
Comment (AbdulrhmnGhanem) [inline .vscode/tasks.json:26]: new line

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+    "azureFunctions.deploySubpath": ".",
+    "azureFunctions.scmDoBuildDuringDeployment": true,
+    "azureFunctions.pythonVenv": ".venv",
+    "azureFunctions.projectLanguage": "Python",
+    "azureFunctions.projectRuntime": "~3",
+    "debug.internalConsoleOptions": "neverOpen",
+    "python.formatting.provider": "black",
+    ]
+}
```
Comment (AbdulrhmnGhanem) [inline .vscode/settings.json:10]: new line

File: .vscode/launch.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+    "configurations": [
+        {
+            "name": "Attach to Python Functions",
+            "type": "python",
+            "request": "attach",
+            "port": 9091,
+            "preLaunchTask": "func: host start"
+        }
+    ]
+}
```
Comment (AbdulrhmnGhanem) [inline .vscode/launch.json:12]: new line

File: .funcignore [hunk 6]
```diff
@@ -0,0 +1,5 @@
+.git*
+.vscode
+local.settings.json
+test
+.venv
```
Comment (AbdulrhmnGhanem) [inline .funcignore:5]: new line

**Review by AbdulrhmnGhanem — CHANGES_REQUESTED** [review]

File: HttpTrigger1/main.py (hunk 2 above)
Comment (nooreldeensalah) [reply HttpTrigger1/main.py:14]: This argument specifies where should I clone the required branch, which contains `database.db`, Should I put it into a variable and use string interpolation to subsitute it?

File: HttpTrigger1/models.py [hunk 7]
```diff
@@ -0,0 +1,68 @@
+import os
+from sqlalchemy import Column, Integer, String, DateTime, ForeignKey, create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker, relationship
+
+database_location = "gpages/database.db"
```
Comment (AbdulrhmnGhanem) [inline HttpTrigger1/models.py:6]: import this one to keep both the model and the functions trigger in sync

**Review by AbdulrhmnGhanem — CHANGES_REQUESTED** [review]

**Review by AbdulrhmnGhanem — APPROVED** [review]

**Pull request merged**